tests/tests/cpld_control_c96_p8

```diff
@@ -1,8 +1,8 @@
 #
-#  cpld_control C96 P8 test
+#  cpld_control C96 P8 aero test
```

Comment:
Maybe just to keep it as "cpld_control C96 P8 test" as P8 will run with aerosol?